serverless/sql-databases/how-to/connect-to-a-database.mdx

```diff
@@ -77,4 +121,6 @@ Use the connection parameters (host, port, user, password, and database name) pr
   Connection methods may vary depending on the framework or library used in your application. Refer to the corresponding official documentation for more information.
```

Comment:
We can also point here to the documentation about SSL/TLS: https://www.scaleway.com/en/docs/serverless/sql-databases/api-cli/secure-connection-ssl-tls/
It's about SSL/TLS but also provide code snippets for many clients and languages that could help.